Notebooks/transcript_dataframe_builder.ipynb

```diff
@@ -0,0 +1,1264 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "***\n",
+    "\n",
+    "<center><h1>Transcript Data Frame Builder Notebook <center><h1>\n",
+    "\n",
+    "***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Goals of this notebook:\n",
+    "    \n",
+    "* Connect to one of our local council's database and filestore.\n",
+    "* Retrieve all available meeting transcripts including their meta data for the corresponding council.\n",
+    "* Build a dataframe from all of this for later analysis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load relevant libraries for the task"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 40,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## For connecting to the database and filestore\n",
+    "import fireo\n",
+    "from google.auth.credentials import AnonymousCredentials\n",
+    "from google.cloud.firestore import Client\n",
+    "from gcsfs import GCSFileSystem\n",
+    "\n",
+    "## For calling cdp database and pipeline functions\n",
+    "from cdp_backend.database import models as db_models\n",
+    "#from cdp_backend.pipeline.transcript_model import Transcript\n",
```

Comment:
Curious why you _arent_ using the `Transcript` object? Don't like it?

Reply:
Thats a good question lol. I don't have a good excuse aside from I  got stuck with it a while back but I will revisit it.